src/plugins/intel_cpu/src/exec_network.cpp

```diff
@@ -177,12 +176,18 @@ ExecNetwork::GraphGuard::Lock ExecNetwork::GetGraph() const {
         std::exception_ptr exception;
         auto makeGraph = [&] {
             try {
+                RuntimeEnv::Ptr rtenv;
```

Comment:
Maybe `GraphContext` name better fits the meaning?

Reply:
Good idea, changed